### PR TITLE
Document error codes

### PR DESCRIPTION
**Pull request by rodrigoborgesdeoliveira**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -367,48 +367,312 @@ export class APIException extends Error {
     }
 }
 
+/**
+ * Error codes that App Store Server API responses return.
+ * 
+ * {@link https://developer.apple.com/documentation/appstoreserverapi/error_codes Error Codes}
```
Comment (alexanderjordanbaker) [inline index.ts:373]: Please update this to use a lowercase c to match the documentation

**Comment by alexanderjordanbaker** [comment]
@rodrigoborgesdeoliveira Thank you for the PR, I've left one minor comment. Also your PR will likely conflict with the new error codes added in https://github.com/apple/app-store-server-library-node/pull/27

File: index.ts (hunk 1 above)
Comment (rodrigoborgesdeoliveira) [reply index.ts]: Thanks. I have updated this.

**Comment by rodrigoborgesdeoliveira** [comment]
Thank you for the heads up on the new error codes, @alexanderjordanbaker. I have resolved the conflicts and documented the new error codes as well. Please let me know if there are any other pending issues.

**Review by alexanderjordanbaker — APPROVED** [review]

**Comment by alexanderjordanbaker** [comment]
Apologies for the delay, this looks good, thank you!

**Pull request merged**